recipes/qpoases/all/test_package/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package CXX)
 
+include("${CMAKE_BINARY_DIR}/conanbuild.info.cmake")
```

Comment:
```suggestion
include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```

Reply:
Sorry for the typo.

Reply:
haha, didn't look either, it always takes two :D